app/main/posts/views/post-view-data.directive.js

```diff
@@ -145,7 +145,7 @@ function PostViewDataController(
         // Otherwise, go get posts just normal
         if ($scope.selectedPost.post) {
             let query = PostFilters.getQueryParams($scope.filters);
-            query.created_before = $scope.selectedPost.post.created;
+            query.created_before_by_id = $scope.selectedPost.post.id;
```

Comment:
Can we add a comment above this line, something like:
// As some posts may have their created date redacted by hide_time
// we instead use the Post Id to allow the API to retrieve the date itself